src/server/mappings/lib/get_types.js

```diff
@@ -5,5 +5,5 @@
  *  @return {Array<string>}
  */
 export function getTypes(mappings) {
-  return Object.keys(mappings);
+  return Object.keys(mappings).filter(type => !type.match(/^_default_$/));
```

Comment:
A string comparison would probably be better here.

```
return Object.keys(mappings).filter(type => type !== '_default_');
```

Reply:
sure thing.  pushed.